selfdrive/car/hyundai/interface.py

```diff
@@ -189,6 +189,10 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long):
       ret.mass = 4095.8 * CV.LB_TO_KG + STD_CARGO_KG # weight from EX and above trims, average of FWD and AWD versions (EX, X-Line EX AWD, SX, SX Pestige, X-Line SX Prestige AWD)
       ret.wheelbase = 2.81
       ret.steerRatio = 13.27 # steering ratio according to Kia News https://www.kiamedia.com/us/en/models/sorento-phev/2022/specifications
+    elif candidate == CAR.KIA_NIRO_HEV_2ND_GEN:
+      ret.mass = 3247.0 * CV.LB_TO_KG + STD_CARGO_KG # weight from EX and above trims
+      ret.wheelbase = 2.72
+      ret.steerRatio = 13.7 # steering ratio according to Kia News https://www.kiamedia.com/us/en/models/niro/2023/specifications
```

Comment:
better to use what `paramsd` converges to. just pull the value from `liveParameters.steerRatio` after a good amount of driving has been done